esmvaltool/recipes/recipe_insurance_risk_index_wp7.yml

```diff
@@ -0,0 +1,59 @@
+###############################################################################
+# namelist_insurance_risk_index.yml
+#
+# Description
+# Namelist for computing the standardized ETCCDI indices which are used in
+#the Actuaries Climate Risk Index.
+
+#
+# Authors
+# Alasdair Hunter (BSC, Spain)
+# Nicolau Manubens (BSC, Spain)
+# Louis-Philippe  (BSC, Spain)
+#
+# Project
+# C3S Magic
+#
+# Reference
+# http://actuariesclimateindex.org/home/
+#
+#
+# This namelist is part of the ESMValTool
+###############################################################################
+---
+datasets:
+  #- {dataset: bcc-csm1-1, type: exp, project: CMIP5,  mip: Amon,  exp: historical,  ensemble: r1i1p1,  start_year: 1981,  end_year: 2000}
+  #- {dataset: bcc-csm1-1, type: exp, project: CMIP5,  mip: Amon,  exp: historical,  ensemble: r1i1p1,  start_year: 2000,  end_year: 2005}
+  - {dataset: IPSL-CM5A-LR, type: exp, project: CMIP5,  mip: day,  exp: historical,  ensemble: r1i1p1,  start_year: 1971,  end_year: 2000}
+  - {dataset: IPSL-CM5A-LR, type: exp, project: CMIP5,  mip: day,  exp: rcp85,  ensemble: r1i1p1,  start_year: 2020,  end_year: 2040}
+
+latlon_roi: (/latmin,latmax,lonmin,lonmax/)
+preprocessors:
+  preproc:
+    regrid:
+      target_grid: IPSL-CM5A-LR
+      scheme: linear
+    mask_fillvalues:
+      threshold_fraction: 0.95
+    extract_region:
+      start_longitude: -60
+      end_longitude: 40
+      start_latitude: 30
+      end_latitude: 70
+
+
+diagnostics:
+  risk_index:
+    description: Calculate insurance risk indices.
+    variables:
+      tasmax:
+        preprocessor: preproc
+        mip: day
+        field: T2M
+
+    scripts:
+      main:
+        script: magic_bsc/risk_index.r #(in diag_scripts)
+
+        metric: t90p # t10p, t90p, cdd, rx5day or Wx
```

Comment:
If I run this recipe with setting `metric: cdd`, the script crashes with the following log message:
```
Loading required package: maps

Attaching package: ‘s2dverification’

The following object is masked from ‘package:base’:

    Filter


Attaching package: ‘ClimProjDiags’

The following object is masked from ‘package:s2dverification’:

    Subset

Warning message:
In dir.create(run_dir, recursive = TRUE) :
  '/group_workspaces/jasmin2/cp4cds1/magic-test/tmp-bandela/MAGIC_BSC/output_dir/recipe_insurance_risk_index_wp7_metric_cdd_20190124_143952/run/risk_index/main' already exists
 num [1:366, 1, 1, 1:36, 1:16] 294 295 294 294 295 ...
Warning message:
In Climdex(data = historical_data, calendar = calendar, metric = metric[m],  :
  Parameter 'threshold' haven't be used when computing metric 'cdd' or 'rx5day'.
[1] "Attribute projection from climatological data is saved and, if it's correct, it can be added to the final output: NULL"
Error in Rsx_nc4_put_vara_double: NetCDF: Numeric conversion not representable
Error in ncvar_put(file, defdata, data) : 
  C function Rsx_nc4_put_vara_double returned error
In addition: Warning message:
In Climdex(data = projection_data, metric = metric[m], calendar = calendar,  :
  Parameter 'threshold' haven't be used when computing metric 'cdd' or 'rx5day'.
Execution halted
```